File: server/CommonLayer/DTOs/DTOReturnLoginUserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CommonLayer.Enums;

namespace CommonLayer.DTOs
{
    public class DTOReturnLoginUserData
    {
        public string Name { get; set; } = string.Empty;
        public string Surname { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Telephone { get; set; } = string.Empty;
        public DateTime DateOfBirth { get; set; }
        public RoleType Role { get; set; }
    }
}

File: client/src/interfaces/DTOReturnLoginUserData.ts
import type { RoleType } from "../enums/RoleType";

export interface DTOReturnLoginUserData {
  name: string;
  surname: string;
  email: string;
  telephone: string;
  dateOfBirth: string; // ISO string (DateTime na backendu)
  role: RoleType;
}